pkgs/servers/search/zinc/default.nix

```
@@ -0,0 +1,70 @@
+{ lib, stdenv, fetchFromGitHub, pkgs, buildGoModule }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, buildGoModule }:
```
Please do not use pkgs here